STYLEGUIDE.md

```diff
@@ -146,27 +146,37 @@ export const Alert = ({
   ...
 ```
 
-### Import individual icons directly from the cjs module
+### Import icons
```

Comment:
I think this whole section can be removed now that we have it in the contributing guide.

Reply:
Removing it sounds good. But before removing it, to confirm, would it be good to mention importing icons from `utils` in `CONTRIBUTING.md` but recommend users not to import individually from `utils/icons` in `STYLEGUIDE.md` ([L149-158](https://github.com/iTwin/iTwinUI-react/pull/991/files#diff-18b19378fcb4038a2ce14f3ef8dd8ffa0bacd8f4e795cef51d08f3178d7e946cR149-R158))?

Reply:
no need. if we see anyone doing it like that in the future, we can reconsider

Reply:
**[DONE]** Sounds good, removed